src/components/data-processing/DataIntegration/DataIntegration.jsx

```diff
@@ -64,7 +63,7 @@ const DataIntegration = (props) => {
     },
     frequency: {
       title: 'Frequency plot',
-      plotUuid: generateDataProcessingPlotUuid(null, filterName, 1),
+      plotUuid: 'dataIntegrationFrequency',
```

Comment:
Why is this plot different and does not require a plotUuid with the ordinal number?

Reply:
relating to this and [this](https://github.com/biomage-ltd/ui/pull/173#discussion_r602149034).
We are not receiving plot data for the frequency plot, it is being generated by looking into the cellSets data in a similar way that the frequency plot in the "plots and tables" module does, so we don't need that special plotUuid generator (that we aren't receiving a plot for anyways).